ui/scripts/start-vault.js

```diff
@@ -42,7 +42,8 @@ readline
     if (root && unseal) {
       fs.writeFile(
         path.join(process.cwd(), 'tests/helpers/vault-keys.js'),
-        `export default ${JSON.stringify({ unseal, root }, null, 2)}`
+        `export default ${JSON.stringify({ unseal, root }, null, 2)}`,
+        err => if (err) throw err;
```

Comment:
looks like this is erroring in travis

maybe it doesn't like the compact version?